SharpBucket/V2/EndPoints/RepositoriesEndPoint.cs

```diff
@@ -22,24 +22,37 @@ public RepositoriesEndPoint(SharpBucketV2 sharpBucketV2)
         {
         }
 
+        
```

Comment:
A useless empty line
```suggestion
```

Maybe we should add something like stylecop to that project one day ?

Reply:
I actually added that line on purpose, to separate the constructor from the instance methods, but I'm happy to take it out.